husarion_ugv_manager/test/plugins/condition/test_check_bool_msg.cpp

```diff
@@ -71,7 +71,7 @@ TEST_F(TestCheckBoolMsg, NoMessageArrived)
   ASSERT_NO_THROW({ CreateTree(PLUGIN, input); });
 
   auto & tree = GetTree();
-  auto status = tree.tickWhileRunning();
+  auto status = tree.tickOnce();
   EXPECT_EQ(status, BT::NodeStatus::FAILURE);
```

Comment:
Why are we changing this. Should work the same but ensures that the tree has fully executed. Applies to other bt tests